source/cellular_pktio.c

```diff
@@ -1191,7 +1189,7 @@ static uint32_t _handleRxDataEvent( CellularContext_t * pContext,
         {
             /* Add the null terminated char to the end of pLine. */
             pLine[ bytesLeft ] = '\0';
-            _handleAllReceived( pContext, ppAtResp, pLine, bytesLeft );
+            _handleAllReceived( pContext, &pContext->pAtCmdResp, pLine, bytesLeft );
```

Comment:
```suggestion
            _handleAllReceived( pContext, &( pContext->pAtCmdResp ), pLine, bytesLeft );
```

Reply:
Address allo the operator precedence in this commit https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface/pull/165/commits/82204b8964fc9835dac94b6fb1259c6822ec3d8e